content/composer/overview.md

```diff
@@ -8,23 +8,25 @@ weight: 200
 ---
 
 
-## Introduction
+## Introduction##
 
-The composer is a graphical editor for creating blueprint YAML files dynamically using a modern drag and drop interface. <br />
-The composer enables topology modeling for complex applications while providing means to add relevant lifecycle operations implementation via external plugins and scripts.  <br />
+Cloudify Composer is a graphical editor for creating blueprint YAML files dynamically, using a modern drag and drop interface.<br />
+Cloudify Composer enables topology modeling for complex applications while providing the ability to add relevant lifecycle operations implementation via external plugins and scripts.  Cloudify Composer is a standalone application that is installed separately from Cloudify Manager.<br />
 
-Among its draggable components you'll find platform & network items (such as 'compute' node, 'database', 'web server' etc..) and you'll be able to add your own custom node types components as well as custom plugins and new interfaces. <br />
+Among its draggable components are platform and network items including 'compute' node, 'database', and 'web server' items, and so on. You can also add your own custom node type components, as well as custom plugins and new interfaces. <br />
 
-The generated result will be a downloadable tgz file containing:  <br />
+The generated result is a downloadable `tgz` file containing:  <br />
 
-- **blueprint.yaml** file which provides a TOSCA based description for the application topology and its lifecycle management. <br />
-- **Custom types and plugins** (if added by the user) and other resources the blueprint.yaml depends on.
+- The `blueprint.yaml` file, which provides a TOSCA-based description for the application topology and its lifecycle management. <br />
+- Optionally, *Custom types* and *plugins* and other resources that the `blueprint.yaml` depends on.
```

Comment:
"depends on" is probably not the right choice of words. It's more correct to say that the blueprint contains operation mappings and resource references to all kinds of scripts and config templates.